app/services/query-builder.js

```diff
@@ -76,7 +76,7 @@ export const associationsQuery = ({ index, page, params, size }) => {
     } else if (params.status === 'Verlopen') {
       addFilter(
         ':query:recognitions.validityPeriod',
-        `(NOT ((recognitions.validityPeriod.startTime:<=${today}) AND (recognitions.validityPeriod.endTime:>=${today})))`,
+        `recognitions.validityPeriod.endTime:<${today}`,
```

Comment:
See if you can change the filter as follow: 
If association has only expired recognitions -> expired
If it has an active recognitions and also expired recognitions -> recognized
If it has a future recognition and also an active recognitions -> recognized
Only future -> nothing
Only active -> recognized